src/mpid/ch4/netmod/ofi/ofi_am.h

```diff
@@ -123,6 +123,11 @@ static inline size_t MPIDI_NM_am_hdr_max_sz(void)
     return MPL_MIN(max_shortsend, max_representable);
 }
 
+static inline size_t MPIDI_NM_am_eager_limit(void)
+{
+    return MPIDI_OFI_DEFAULT_SHORT_SEND_SIZE - sizeof(MPIDI_OFI_am_header_t);
+}
+
```

Comment:
This doesn't take into account d061878934d3 and 064dacbe7207ad1f7288a79b183f2f229e2102b7 that adds a CVAR to overwrite this value. Probably all versions of this function need to check whether this CVAR is specified.

Reply:
I wasn't aware of that CVAR, thanks for pointing it out.

Ken convinced me that the eager limit should be dictated by the lower-layer -- netmod or shmmod -- because the lower layer knows what makes sense for their mechanism. In what scenario that user desire to set eager limit regardless of lower-layer transport?

Reply:
I don't have a problem with that CVAR being a nm/shm level decision. I just want to make sure the override exists. It's really only used for testing, but tweaking the eager/rndv threshold is also one of the classic examples of an "MPI Tuning Knob" that people want to see.

Reply:
Sounds good. I'll update it to honor that CVAR.

Reply:
Added a commit to apply this CVAR at MPIDIG layer.